assets/js/lib/api/wanda.js

```diff
@@ -1,16 +1,14 @@
-import axios from 'axios';
+import { networkClient } from '@lib/network';
 
 const baseURL = process.env.WANDA_URL;
 
-export const wandaClient = axios.create({
-  baseURL,
-});
-
 export const getExecutionResult = (executionID) =>
-  wandaClient.get(`/api/checks/executions/${executionID}`);
+  networkClient.get(`/api/checks/executions/${executionID}`, { baseURL });
```

Comment:
Instead of having to pass the `baseURL` on each call, we could export a function from `@lib/network` that returns an authenticated client on a given `baseURL` (aka service?)

So we'd need to only change how we build the `wandaClient`
```js
export const wandaClient = createAuthenticatedClient({ baseURL });
```
wdyt?

Reply:
The authenticated client should be a singleton for all the application, like now, we could achieve that with a little of closure magic ofc, but no hard feelings

Having a shared object in the Wanda client with the base URL option and spreading it through the calls like @dottorblaster suggested in the previous pr should be done

Reply:
Addressed #1132 comments